unified-scheduler-pool/src/lib.rs

```diff
@@ -3686,7 +3686,7 @@ mod tests {
         poh_recorder
             .write()
             .unwrap()
```

Comment:
need `poh_recorder.track_transaction_indexes();` here?

Reply:
🤷‍♂️ should have for consistency, but if the test doesn't need the indexes then we shouldn't have them